packages/discussions/src/types.ts

```diff
@@ -98,6 +98,7 @@ export enum PostStatus {
 export enum DiscussionType {
   GROUP = "group",
   CONTENT = "content",
+  BOARD = "board",
```

Comment:
I'm curious if this is explicitly defined anywhere in the A.C.? I _believe_ this is desirable, so that we can separate discussion board posts from our existing content posts in discussions panel. Andrew's vision is that an idea board can pertain to other types of items (e.g. Hub Site, PDF, Event, etc), not just items with type=discussion. In the future we'll be tackling forums, so that would probably follow this same convention to isolate those posts from board & content.

Reply:
@rweber-esri From a functionality standpoint, this is mostly to make it easier to refactor [arcgis-hub-discussions-options](https://github.com/ArcGIS/opendata-ui/blob/3213f14f287e7b6cdf447f26e660bf8979bcfbc8/packages/hub-components/src/components/arcgis-hub-discussions/components/arcgis-hub-discussions-options/arcgis-hub-discussions-options.tsx#LL35C28-L35C28), which uses DiscussionType to adjust the strings. But I figured expanding this would also be a good idea for the reasons you mention.